test/ui/ui-tests-user-id.js

```diff
@@ -0,0 +1,12 @@
+const puppeteer = require('puppeteer');
+const screenshotPath = `../results/ui-test-results/users-id/ui-user-user-id.png`;
```

Comment:
Are we just keeping the screenshot for the current build or will we want to keep a record of all previous screenshots? Because when you upload this, it will (I think) overwrite the current ui-user-user-id.png screenshot.